MaintenanceExam/PageObjects/LoginPage.cs

```diff
@@ -7,6 +7,10 @@ public class LoginPage : BasePageLocal
         private readonly By _logInLocator = By.XPath("//button[contains(.,'Log In')]");
         private readonly By _passwordLocator = By.Id("Password");
         private readonly By _usernameLocator = By.Id("Username");
+        private readonly By _searchBoxLocator = By.CssSelector("div.search-field input");
```

Comment:
I'm not sure what all these locators are for? Did you forget to remove them or ?